lib/fast_jsonapi/object_serializer.rb

```diff
@@ -221,7 +221,7 @@ def create_relationship_hash(base_key, relationship_type, options, block)
       end
 
       def compute_serializer_name(serializer_key)
-        return serializer_key unless serializer_key.is_a? Symbol
+        return serializer_key.to_s.to_sym unless serializer_key.is_a? Symbol
```

Comment:
The output of this method is always used as
`@relationships_to_serialize[item][:serializer].to_s.constantize` 

so why not change this method to simply return a string? Not sure what's the point of converting to symbol then back to string.

Reply:
Good point. I think it would just make sense always return a `constantize` serializer Class instead of a String or Symbol. It cuts the conversation.

So this would look like this:
```
      def compute_serializer_name(serializer_key)
        return serializer_key.to_s.constantize unless serializer_key.is_a? Symbol
        namespace = self.name.gsub(/()?\w+Serializer$/, '')
        serializer_name = serializer_key.to_s.classify + 'Serializer'
        return (namespace + serializer_name).constantize if namespace.present?
        (serializer_key.to_s.classify + 'Serializer').constantize
      end
```

By doing this we can make the output be:
```
@relationships_to_serialize[item][:serializer]
```

Thoughts @zinosama ?

Reply:
If the serializer class doesn't exist, this approach will display a more accurate/upstream source of error - thus I think it's a better solution.